README.md

```diff
@@ -145,8 +145,17 @@ then next you will create a `BUILD.bazel` file in your project root containing:
 ```python
```

Comment:
> other workspaces can't depend on yours without knowing how to install your deps

Not sure bazel-managed deps is going to fix this since if your workspace uses yarn_install and created an external repository `@npm` then another workspace isn't going to see `@npm` from your workspace (and if it did it would conflict with `@npm` in the downstream workspace)

Reply:
Updated the README and added a **Bazel-managed vs self-managed dependencies** section before going into the details of both. I also moved the **Using Bazel-managed dependencies*** above **Using self-managed dependencies** since it is the preferred approach.